### DISCUSSION TITLE
Bind drag trigger to only a title area?

### DISCUSSION THREAD

**Discussion by joshuakoh1** [react-grid-layout/react-grid-layout#1692] [Q&A] [answered]
Hi,

The draggable trigger is interfering with the interactive components within the grid. 

E.g. Trying to select an area on a plotly chart also triggers the drag which makes it impossible to use. 

Is there a way to bind the drag control only to a specific portion of the grid, perhaps maybe with a title bar per grid?

Thanks

**Comment by joshuakoh1** [comment, marked as answer]
Solved with draggableCancel and draggableHandle. I was putting them under the tiles instead of the grid. That's why it didn't work before.

**Comment by Hardik-Rana-Silicon-IT-Hub** [comment]
Hi @joshuakoh1 

```draggableCancel``` for cancel draggable.

```draggableHandle``` for particular Handle From element. it will target a class and it will automatically disable drag on full element. so, you can drag of only targeted element.
